src/libraries/System.IO.Compression/src/System/IO/Compression/ZLibStream.cs

```diff
@@ -45,6 +45,18 @@ public ZLibStream(Stream stream, CompressionLevel compressionLevel, bool leaveOp
             _deflateStream = new DeflateStream(stream, compressionLevel, leaveOpen, ZLibNative.ZLib_DefaultWindowBits);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ZLibStream"/> class by using the specified stream, compression options, and whether to leave the <paramref name="stream"/> open.
```

Comment:
For consistency with the other similar summaries:
```suggestion
        /// Initializes a new instance of the <see cref="ZLibStream"/> class by using the specified stream, compression options, and optionally leaves the stream open.
```